ash/src/extensions/ext/tooling_info.rs

```diff
@@ -30,7 +30,7 @@ impl ToolingInfo {
         &self,
         physical_device: vk::PhysicalDevice,
     ) -> VkResult<Vec<vk::PhysicalDeviceToolPropertiesEXT>> {
-        read_into_uninitialized_vector(|count, data| {
+        read_into_defaulted_vector(|count, data| {
```

Comment:
commit message seems off.

Reply:
Seems correct to me; solving this problem is exactly why `read_into_defaulted_vector` exists.